chef-dk.gemspec

```diff
@@ -53,7 +53,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "chef-provisioning", "~> 1.2"
 
-  gem.add_development_dependency "github_changelog_generator"
+  gem.add_development_dependency "github_changelog_generator", "!= 1.10.2", "!= 1.10.3", "!= 1.11.0"
```

Comment:
I see this is already merged but if you have a moment to add a comment linking to the upstream issue, that'd be nice so we will know when we can remove this.